### ISSUE TITLE
Compile DOtherSide statically with mingw and build nim-status-client for windows with mingw

### ISSUE THREAD

**Issue by richard-ramos** [status-im/nim-status-client#90]
https://github.com/status-im/nim-status-client/blob/windows-2/Makefile can be used as a template for this.

Notice that there are some changes:

- DOtherside has a different target:
```
$(MAKE) DOtherSideStatic
```

- Different flags in `build-linux`
```
$(ENV_SCRIPT) nim c -d:ssl -L:$(STATUSGO) \
	-L:$(DOTHERSIDE) \
	-L:-L/home/richard/Qt/5.14.2/gcc_64/lib \
	-L:-lQt5QuickControls2 -L:-lQt5Quick -L:-lQt5QmlModels -L:-lQt5Qml -L:-lQt5Network -L:-lQt5Widgets -L:-lQt5Gui -L:-lQt5Core \
	-L:-lstdc++ \
	-L:-lm \
	$(NIM_PARAMS) --outdir:./bin src/nim_status_client.nim
```

- A new section in the makefile for Windows:  `build-windows`
https://github.com/status-im/nim-status-client/blob/windows-2/Makefile#L160-L189

This task is blocked by: https://github.com/status-im/dotherside/issues/3

**Comment by michaelsbradleyjr** [comment]
I'm working on this currently, didn't realize we had an open issue for it, but after seeing it I've assigned myself.

**Comment by michaelsbradleyjr** [comment]
Closed by #589. Note: we ended up needing to build DOtherSide as a DLL on Windows. That's because the Qt's mingw toolchain for Windows lacks the WebEngine and it was simpler to build a DLL with the msvc toolchain than doing a static build.
